src/NSwag.Sample.NET60Minimal/Program.cs

```diff
@@ -20,4 +11,21 @@
 app.MapGet("/", (Func<string>)(() => "Hello World!"));
 app.MapGet("/sum/{a}/{b}", (Func<int, int, int>)((a, b) => a + b));
 
-app.Run();
\ No newline at end of file
+app.Run();
+
+public partial class Program
+{
+    public static WebApplication BuildHost(string[] args)
+    {
+        var builder = WebApplication.CreateBuilder(args);
+
+        builder.Services.AddEndpointsApiExplorer();
+        builder.Services.AddOpenApiDocument(settings =>
+        {
+            settings.Title = "Minimal API";
```

Comment:
The flag on this doesn't make sense, the new Startup-less program.cs is used for both controller-based and minimal API code templates. 

I don't have a name in mind, maybe this name fix can wait for a future release when the approach settles in with the community?